src/odemis/gui/cont/tabs.py

```diff
@@ -6618,6 +6618,7 @@ def _onAlignMode(self, mode):
         elif mode == "center-align":
             self.tab_data_model.focussedView.value = self.panel.vp_align_center.view
             self._ccd_stream.should_update.value = True
+            # TODO: allow to reset background. Also reset automatically when switching between modes
```

Comment:
Now the code allows to reset => change TODO to just "automatically reset" (or "use a different stream"?)

Reply:
I think "automatically reset" is fine is this case although I am not completely sure how both options differ. Where is the new code that allows to reset to be found?

Reply:
The new code is the `self.panel.btn_bkg_acquire.Enable(True)`. It shows the "Acquire background" button if there is no background yet, and a "Remove background" if the background is already acquired.

Reply:
Alright, could you in that case delete the TODO comment (its done right) and add a comment to line 6638 describing the choice to either remove or acquire the background based on the current state?